app/services/rapid_ready_for_decision/processor_selector.rb

```diff
@@ -0,0 +1,48 @@
+# frozen_string_literal: true
+
+module RapidReadyForDecision
+  class ProcessorSelector
+    def initialize(form526_submission)
+      @form526_submission = form526_submission
+    end
+
+    def processor_class
+      return nil unless rrd_enabled?
+
+      if hypertension_enabled? && single_issue_claim_applicable?(DiagnosticCodes::HYPERTENSION)
```

Comment:
I wonder if this condition can be wrapped up more neatly in a named method, if there's a good term...